### PR TITLE
DB-1812: add template Consentric SystemAddon

### PR DESCRIPTION
**Pull request by alver-cliqz**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -49,8 +49,10 @@ properties([
                 name: 'CQZ_EXTENSION_URL'),
         string(defaultValue: "5B0571C810B2BC947DE61ADCE8512CEA605A5625",
                 name: "CQZ_CERT_NAME"),
-        string(defaultValue: 's3://cdncliqz/update/browser/https-everywhere/https-everywhere@cliqz.com-5.2.17-browser-signed.xpi',
+        string(defaultValue: 's3://cdncliqz/update/browser_beta/https-everywhere/https-everywhere@cliqz.com-2018.6.21-browser_beta-signed.xpi',
```
Comment (luciancor) [inline Jenkinsfile]: we should use the release link here

File: mozilla-release/browser/app/profile/firefox.js [hunk 2]
```diff
@@ -81,7 +81,7 @@ pref("extensions.webextensions.remote", true);
 #endif
 
 // Extensions that should not be flagged as legacy in about:addons
-pref("extensions.legacy.exceptions", "cliqz@cliqz.com,https-everywhere@cliqz.com,testpilot@cliqz.com,@testpilot-containers,jid1-NeEaf3sAHdKHPA@jetpack,@activity-streams,pulse@mozilla.com,@testpilot-addon,@min-vid,tabcentertest1@mozilla.com,snoozetabs@mozilla.com,speaktome@mozilla.com,hoverpad@mozilla.com");
+pref("extensions.legacy.exceptions", "cliqz@cliqz.com,https-everywhere@cliqz.com,gdprtool@cliqz.com,testpilot@cliqz.com,@testpilot-containers,jid1-NeEaf3sAHdKHPA@jetpack,@activity-streams,pulse@mozilla.com,@testpilot-addon,@min-vid,tabcentertest1@mozilla.com,snoozetabs@mozilla.com,speaktome@mozilla.com,hoverpad@mozilla.com");
```
Comment (luciancor) [inline mozilla-release/browser/app/profile/firefox.js]: gdrptool is not a legacy extension

File: Jenkinsfile (hunk 1 above)
Comment (alver-cliqz) [reply Jenkinsfile]: no, this is file for PR build

Comment (luciancor) [reply Jenkinsfile]: 👍

**Pull request merged**